src/scikit_build_core/builder/wheel_tag.py

```diff
@@ -37,7 +37,7 @@ def compute_best(
         pyvers = [interp]
 
         if sys.platform.startswith("win") and archs:
-            plats = [x.replace("-", "_") for x in archs]
```

Comment:
I made this change because mypy (via pre-commit) was complaining at me that `plats` needs to be a `list[str]` (which is what the `archs` field of the class will eventually be anyway. I couldn't commit without making that change, so I am surprised that you haven't seen the same error changing it back.

Reply:
Hmm, I see that. I was just trying to minimize the diff, and nothing else changed in this file. I think it's better if the type remains consistent. Were you using pre-commit & the same version of mypy?

Reply:
Ahh, this will only hit if you are running mypy on Windows! I can verify I can replicate it with `args: [--platform=win32]`.

Reply:
Fixed with #217.